perf_tests/perf_testing_basic.md

```diff
@@ -0,0 +1,39 @@
+# Doc: demo system
+
+Created by: Deepika Upadhyay
+Created time: November 7, 2023 5:33 PM
+
+# Performance Testing in Demo System
+
+For performance testing, we follow the Koor Knowledge Base basically, you can find it [here](https://kb.koor.tech/knowledge/ceph/benchmarking/)
+
+Once you have gone through the guide, you can use the following cheatsheet for demo environment perf testing Here’s a short summary of commands used 
+
+## Disk Performance Commands used with fio
+
+commonly used commands have been adapted to this script.
+
+[https://github.com/koor-tech/demo-gitops/blob/main/perf_tests/test-fio.sh](https://github.com/koor-tech/demo-gitops/blob/main/perf_tests/test-fio.sh) 
+
+## Network Performance using Ancientt:
+
+This step should be performed after you have installed the Kubernetes cluster, we use [ancientt](https://github.com/galexrt/ancientt) tool, which uses iperf3 to perform cross node network testing.
+
+- please refer the installation guide to install ancientt on you development enviroment,
```

Comment:
Let's link to the install guide of `ancientt` so it is easy to run the tests

Reply:
good point, updated, thanks!